pkg/api/validation/validation.go

```diff
@@ -687,11 +687,10 @@ func ValidatePodStatusUpdate(newPod, oldPod *api.Pod) errs.ValidationErrorList {
 
 	allErrs = append(allErrs, ValidateObjectMetaUpdate(&oldPod.ObjectMeta, &newPod.ObjectMeta).Prefix("metadata")...)
 
-	// TODO: allow change when bindings are properly decoupled from pods
```

Comment:
When would this happen?  Kubelet would have to post a host that is empty to a pod that has a host?